src/main/java/org/spongepowered/mod/registry/SpongeGameRegistry.java

```diff
@@ -279,6 +280,15 @@ public TradeOfferBuilder getTradeOfferBuilder() {
         return Optional.fromNullable((EntityType) this.entityIdToTypeMappings.get(id));
     }
 
+    public Optional<EntityType> getEntity(Class<? extends org.spongepowered.api.entity.Entity> entityClass) {
+        for(EntityType type : getEntities()) {
```

Comment:
Can this just be done by calling `entityClass.getType`?

The method `getType` is inherited from `EntityState`